source/_docs/energy/water.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: "Integrating your water usage"
+description: "Learn how to add information about your water usage to Home Assistant home energy management."
+---
+
+Home Assistant allows you to track your water usage in the home energy management too.
+
+Although water is not strictly "energy", it is still a valuable resource to track and monitor as it is often tightly coupled with energy usage (like gas). Additionally, it can help you reduce your ecological footprint by using less water. 
+
+## Hardware
+
+Home Assistant will need to know the amount of water that is being consumed in order to be able to track it. Several hardware options are available to do this.
+
+We have the following integrations available for existing products that can provide information about water usage:
+
+- [Flo](/integrations/flo)
+- [Flume](/integrations/flume)
+- [HomeWizard Energy](/integrations/homewizard)
+
+If you have a smart water meter that is wirelessly connected to your electricity meter, the following integration could be of help:
+
+- [P1 Monitor](/integrations/p1_monitor) 
```

Comment:
P1 Monitor uses a simple pulse meter coupled to GPIO pins, so I think it falls more into the DIY category than something ready-made and wireless?

Reply:
Klaas is correct. Docs mention soldering:

https://www.ztatz.nl/p1-monitor-watermeter/

Reply:
I've removed the section. Added the P1 Monitor to the DIY paragraph and removed Toon. Toon is almost not used and would only work for a very very small percentage of Dutch users that have a smart water meter (which isn't rolled out anymore at this point).